apis/kubedb/v1alpha2/solr_types.go

```diff
@@ -83,6 +87,15 @@ type SolrSpec struct {
 	// To enable ssl for http layer
 	EnableSSL bool `json:"enableSSL,omitempty"`
 
+	// Client auth need
+	NeedClientAuth bool `json:"needClientAuth,omitempty"`
+
+	// Client auth want
+	WantClientAuth bool `json:"wantClientAuth,omitempty"`
+
```

Comment:
use one api instead of two `ClientAuth string` or remove both if you can pass using env